packages/astro/test/astro-pagination.test.js

```diff
@@ -50,4 +50,16 @@ describe('Pagination', () => {
 			})
 		);
 	});
+
+	it('canonical URLs', async () => {
+		for (const [file, canonicalURL] of [
+			['/posts/red/1/index.html', 'https://mysite.dev/blog/posts/red/1/'],
```

Comment:
The canonical URL was already being added in the test fixture, just needed to add validation for it